app/javascript/controllers/ui/dialog_controller.js

```diff
@@ -31,6 +31,21 @@ export default class UIDialog extends Controller {
     e.preventDefault();
   }
 
+  handleFocusEvent(e) {
+    let target = this.dialogTarget;
+    let shiftPressed = e.shiftKey;
+    // If TAB is pressed
+    if (e.keyCode === 9) {
+      let focusableElements = target.querySelectorAll("a[href], button");
```

Comment:
Non-blocking, but if the value aren't going to be re-declared please use `const`